#!/usr/bin/env python
PACKAGE = "graph_d_exploration"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("ig_threshold", double_t, 0, "A double parameter", 0.45, 0, 1)

exit(gen.generate(PACKAGE, "graph_d_exploration", "informationGain"))
